modules/json/src/smithy4s/http/json/codecs.scala

```diff
@@ -0,0 +1,15 @@
+package smithy4s
+package http.json
+
+import smithy4s.syntax._
+
+final case class codecs(mask: HintMask)
```

Comment:
Interesting API design here (the parameterless and parameterized version of codecs) :+1: